minimint/Cargo.toml

```diff
@@ -16,6 +16,7 @@ hbbft = { git = "https://github.com/fedimint/hbbft", branch = "minimint" }
 hex = "0.4.2"
 itertools = "0.10.0"
 minimint-api = { path = "../minimint-api" }
+minimint-core = { path = "../minimint-core" }
```

Comment:
Can we stop depending on the modules directly this way?

Reply:
done in 1aba4892d28c1654071b1751b793f5b52faa545b